gotocompany/optimus/core/v1beta1/job_spec.proto

```diff
@@ -294,7 +296,9 @@ message ChangeJobNamespaceRequest {
   string new_namespace_name = 4;
 }
 
-message ChangeJobNamespaceResponse {}
+message ChangeJobNamespaceResponse {
+  bool success = 1;
```

Comment:
Please remove this, we should not be sending boolean success field in an api. Return an error in case of any problem. Response from server with `Success = false` does not make any sense.